kernel/libc/koslib/realpath.c

```diff
@@ -26,11 +26,10 @@
  * SUCH DAMAGE.
  */
 
+#include <kos/limits.h>
```

Comment:
Why'd this need to be moved?

Reply:
Looks like he was going from bigger to smaller depending on the length of the includes.  I do this too.